src/main/scala/org/hammerlab/guacamole/likelihood/Likelihood.scala

```diff
@@ -157,7 +157,7 @@ object Likelihood {
             case (allele, alleleFraction) =>
               alleleElementProbabilities(alleleToIndex(allele), ::) * alleleFraction
           }
-          sum( log( sum(alleleRows) )) + math.log(prior(genotype)) * depth
+          sum( log( sum(alleleRows) + 1e-10 )) + math.log(prior(genotype))
```

Comment:
I'm still chewing on this `1e-10`; I notice that removing it allows the `epsilon` error in `LikelihoodSuite` to be `1e-12` instead of `1e-5`. The latter seems unacceptably high for the computations we're doing here.